pootle/core/search/broker.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from . import SearchBackend
+
+import importlib
+
+from django.conf import settings
+
+
+class SearchBroker(SearchBackend):
+    def __init__(self, config_name=[]):
+        self._setup_settings()
+        self._servers = {}
+        if self._settings is None:
+            return
+        for server in self._settings:
+            if config_name == [] or server in config_name:
```

Comment:
When checking if a sequence has elements, use the fact that empty sequences are false (PEP8): `if not config_name or server in config_name:`